### DISCUSSION TITLE
How can I change the scaling of everything?

### DISCUSSION THREAD

**Discussion by moxislut** [JaKooLit/Hyprland-Dots#303] [General]
Hello! I have a fairly weird resolution (2880x1800) and I would like to make everything just a tad bit bigger. I've tried looking at the documentation and I haven't found anything that would hep me with this issue. (Unless I am just being stupid and just overlooked something?)

**Comment by JaKooLit** [comment]
https://wiki.hyprland.org/Configuring/Monitors/

Thats a really weird resolution

SUPER E choose monitor

monitor=,preferred,auto,1 

you can try to put 1.25, or 1.5 or even auto at the end

**Reply by moxislut** [reply]
Thank you so much! I knew it was in the monitor settings somewhere, I just couldn't figure out what. 

In-case you were curious, I am using the ASUS Zenbook 14 OLED

**Reply by JaKooLit** [reply]
yup. Have heard about those crazy resolution on a laptop.. its useless imho.. Its just raising the prices

having myself a 2k res on monitor on 15 inches is OK (kinda).. but I dont scale it. I just increase font size. more crisp...

anything above that, on a laptop, I wont purchase...


anyway, so can I close this?

**Reply by moxislut** [reply]
Also quick question; I don't wanna start another discussion, so hopefully you don't mind me replying here. My Dolphin is being very fucky, and by that I mean it's very VERY laggy and begins to spike my CPU usage. Anything I can do to fix that?

**Reply by JaKooLit** [reply]
> Also quick question; I don't wanna start another discussion, so hopefully you don't mind me replying here. My Dolphin is being very fucky, and by that I mean it's very VERY laggy and begins to spike my CPU usage. Anything I can do to fix that?

no idea bro. I dont use dolphin outside of plasma.. and I have only thunar on my system with hyprland

**Comment by JaKooLit** [comment]
oh no need to close.. its on discussion :)

**Reply by thebadking** [reply]
how do I setup if I have two monitors? I need different scaling for both

**Reply by JaKooLit** [reply]
> how do I setup if I have two monitors? I need different scaling for both

https://wiki.hyprland.org/Configuring/Monitors/

![Screenshot_20240718_072541_Samsung Internet](https://github.com/user-attachments/assets/ee46a6e6-0c3d-4494-9db7-2a1e705e38a6)


the hyprland wiki explained well

so edit Monitors.conf accordingly
